assets/js/specific/ajaxTextObjectRecSearch/ajax-text-search.js

```diff
@@ -1,24 +1,35 @@
-( function( $ ) {
-
-	$( document ).ready( function() {
+pixl8presideExtMultiselect.fn.ajaxSearch = function( $container ) {
```

Comment:
We shouldn't need or be introducing new global variables in our javascript. I would have thought this would make sense as a jquery plugin? i.e. 

```js
( function( $ ){
	$.fn.pixl8PresideExtMultiSelect = function(){
		return $( this ).each( function(){
			let   $selectField     = $( this )
			///...
		} );   
        };
        
        $( "select.custom-select[data-ajax-txt-search=1]" ).pixl8PresideExtMultiSelect();
        
} )( jQuery );
```